### PR TITLE
IBX-8536: Removed deprecations in graphql

### PR DESCRIPTION
**Pull request by glye**
| :ticket: Issue | IBX-8536 |
|----------------|-----------|
#### Additional PRs:
- https://github.com/ibexa/fieldtype-matrix/pull/53
- https://github.com/ibexa/fieldtype-query/pull/31

#### Description:
Resolve deprecations in ibexa/graphql

🟢 https://github.com/ibexa/commerce/pull/1088

### CODE REVIEW

**Review by konradoboza — APPROVED** [review]
Looks okay-ish as long as regressions or E2E GraphQL tests don't expose any missing usage.

**Comment by glye** [comment]
@konradoboza Am looking into getting all tests running. Is there an up to date doc on installing v5? Am struggling with https://ibexa.atlassian.net/wiki/spaces/ENG/pages/5701714/Installation+via+Flex and dev-master / 5.0.x-dev.

File: src/lib/Resolver/DomainContentResolver.php [hunk 1]
```diff
@@ -132,14 +132,6 @@ public function resolveMainUrlAlias(Content $content)
         return isset($aliases[0]->path) ? $aliases[0]->path : null;
     }
 
-    /**
-     * @deprecated since v3.0, use ItemResolver::resolveItemFieldValue() instead.
-     */
-    public function resolveDomainFieldValue(Content $content, $fieldDefinitionIdentifier, $args = null)
```
Comment (ViniTou) [inline src/lib/Resolver/DomainContentResolver.php:138]: Please look at https://github.com/ibexa/graphql/blob/main/src/bundle/Resources/config/services/resolvers.yaml#L35
I think this maybe the only case we have here, but we need to check if other removed methods are not used anywhere.

**Review by ViniTou — CHANGES_REQUESTED** [review]

**Review by konradoboza — APPROVED** [review]

**Review by tbialcz — APPROVED** [review]

**Comment by konradoboza** [comment]
Merging as regressions passed, ref: https://github.com/ibexa/commerce/actions/runs/11477344169/job/31939210332?pr=1088.

**Pull request merged**